{
    "@timestamp": "2025-03-13T12:26:13.978Z",
    "agent": {
        "ephemeral_id": "77c158b8-3388-40b5-8744-32b068f92e8f",
        "id": "72c7c64a-538c-44a6-9d7e-b128b194cd3a",
        "name": "elastic-agent-69763",
        "type": "packetbeat",
        "version": "8.17.3"
    },
    "client": {
        "bytes": 63,
        "ip": "127.0.0.1",
        "port": 37881
    },
    "data_stream": {
        "dataset": "network_traffic.pgsql",
        "namespace": "41189",
        "type": "logs"
    },
    "destination": {
        "bytes": 147,
        "ip": "127.0.0.1",
        "port": 5432
    },
    "ecs": {
        "version": "8.17.0"
    },
    "elastic_agent": {
        "id": "72c7c64a-538c-44a6-9d7e-b128b194cd3a",
        "snapshot": false,
        "version": "8.17.3"
    },
    "event": {
        "agent_id_status": "verified",
        "category": [
            "network"
        ],
        "duration": 2777539,
        "end": "2025-03-13T12:26:13.981Z",
        "ingested": "2025-03-13T12:26:16Z",
        "kind": "event",
        "start": "2025-03-13T12:26:13.978Z",
        "type": [
            "connection",
            "protocol"
        ]
    },
    "host": {
        "architecture": "x86_64",
        "containerized": true,
        "hostname": "elastic-agent-69763",
        "ip": [
            "172.22.0.2",
            "172.18.0.9"
        ],
        "mac": [
            "02-42-AC-12-00-09",
            "02-42-AC-16-00-02"
        ],
        "name": "elastic-agent-69763",
        "os": {
            "family": "",
            "kernel": "5.15.153.1-microsoft-standard-WSL2",
            "name": "Wolfi",
            "platform": "wolfi",
            "type": "linux",
            "version": "20230201"
        }
    },
    "network": {
        "bytes": 210,
        "community_id": "1:bWRv0CI/a87XuiElFcjvxmuWQrY=",
        "direction": "ingress",
        "protocol": "pgsql",
        "transport": "tcp",
        "type": "ipv4"
    },
    "network_traffic": {
        "pgsql": {
            "error_code": "23505",
            "error_message": "duplicate key value violates unique constraint \"unique_a\"",
            "error_severity": "ERROR",
            "method": "INSERT",
            "num_fields": 0,
            "num_rows": 0,
            "query": "insert into test(a, b, c) values('mea5', 'meb5', 'mec5')"
        },
        "status": "Error"
    },
    "related": {
        "ip": [
            "127.0.0.1"
        ]
    },
    "server": {
        "bytes": 147,
        "ip": "127.0.0.1",
        "port": 5432
    },
    "source": {
        "bytes": 63,
        "ip": "127.0.0.1",
        "port": 37881
    }
}
